doc/schema-description.md

```diff
@@ -18,6 +18,7 @@
 |                | crash_reason           | character varying(1024) | No        | Reason for the most reason crash (e.g. app failed the healthcheck)                                                             |
 |                | expire_time            | bigint                  | No        | Unused                                                                                                                         |
 |                | presence               | integer                 | No        | Describes the presence of the cell hosting the ActualLRP. 0 for `Ordinary`, 1 for `Evacuating`, and 2 for `Suspect`.           |
+|                | routable               | boolean                 | No        | True if the ActualLRP is ready to serve traffic, i.e. the LRP has passed any *defined* readiness checks. False otherwise.      |
```

Comment:
Consider adding the note about what happens when there are no defined readiness checks.